doc/fluxing.rst

```diff
@@ -89,20 +89,31 @@ The algorithm options are:
 ++++++
 
 Provide a file to guide the process.  Do this if your changes to
-the defaults are not accommodated by the script inputs.
+the defaults are not accommodated by the script inputs.  This file
+contains a Parameter Block where you can specify ``sensfunc`` parameters.
+For example, if you wish to use the MaunaKea telluric grid with your data,
+you would create a sens file containing::
```

Comment:
I would check how this renders in the html.  The double colon `::` here indicates that what follows is code, so my guess is that the `.. code-block:: ini` code is going to render in the html, and you don't want it to.

Reply:
Fixed.  I'm new to RST, so thanks for catching this error.